netplugin/agent/state_event.go

```diff
@@ -323,8 +307,6 @@ func processServiceLBEvent(netPlugin *plugin.NetPlugin, svcLBCfg *mastercfg.CfgS
 	portSpecList := []core.PortSpec{}
 	portSpec := core.PortSpec{}
 
-	netPlugin.Lock()
-	defer func() { netPlugin.Unlock() }()
```

Comment:
Can you change all of these to `defer netplugin.Unlock()`, please? The `defer func() {}` should be used only when there's more than one line of code to run in defer.